ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils_filtered_tenants.py

```diff
@@ -45,6 +45,36 @@ def _filtered_tenants(self) -> list[dict]:
                     tenant["vrfs"] = self._filtered_vrfs(tenant)
                     filtered_tenants.append(tenant)
 
+        no_vrf_default = all(vrf["name"] != "default" for tenant in filtered_tenants for vrf in tenant["vrfs"])
+        if self.shared_utils.wan_role is not None and no_vrf_default:
+            filtered_tenants.append(
+                {
+                    "name": "WAN_DEFAULT",
+                    "vrfs": [
+                        {
+                            "name": "default",
+                            "vrf_id": 1,
+                            "svis": [],
+                            "l3_interfaces": [],
+                            "bgp_peers": [],
+                            "ipv6_static_routes": [],
+                            "static_routes": [],
+                        }
+                    ],
+                    "l2vlans": [],
+                }
+            )
+        elif self.shared_utils.wan_role:
+            # It is enough to check only the first occurence of default VRF as some other piece of code
+            # checks that if the VRF is in multiple tenants, the configuration is consistent.
+            for tenant in filtered_tenants:
+                if (vrf_default := get_item(tenant["vrfs"], "name", "default")) is None:
+                    continue
+                if "evpn" in vrf_default.get("address_families", ["evpn"]):
+                    if self.shared_utils.underlay_filter_peer_as:
+                        raise AristaAvdError("'underlay_filter_peer_as' cannot be used while there are EVPN services in the default VRF.")
```

Comment:
I'm not sure this is the error you wish to raise here :)
```suggestion
                        raise AristaAvdError("WAN configuration requires EVPN to be enabled for VRF 'default'. Got 'address_families: {vrf_default['address_families']}'.")
```